examples/tabs/tabs-manual.html

```diff
@@ -68,10 +122,11 @@ <h2 id="ex_label">Example</h2>
 
       <section>
         <h2>Accessibility Features</h2>
-        <p>
-          To demonstrate the effects of deleting a tab, the third tab, labeled <q>Joke</q>,
-          can be deleted when it has focus by pressing <kbd>Delete</kbd>.
-        </p>
+        <ul>
+          <li>Since the first focusable element in each <code>tabpanel</code> is a focusable element (e.g. link), the tabpanel does <strong>not</strong> need to set a <code>tabindex</code> value to include it in the tab sequence of the page.</li>
```

Comment:
Just removing redundant word "focusable".

```suggestion
          <li>Since the first element in each <code>tabpanel</code> is a focusable element (e.g. link), the tabpanel does <strong>not</strong> need to set a <code>tabindex</code> value to include it in the tab sequence of the page.</li>
```